<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-4D9NKMKK" failures="1" tests="4" name="TestRunner.Run" time="75.188" errors="0" timestamp="2025-07-14T00:56:52 IST" skipped="0">
  <testcase classname="TestRunner.Run" name="runScenario" time="17.372">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:87)
at org.junit.Assert.fail(Assert.java:96)
at StepDefination.StepDef.added_user_should_be_displayed(StepDef.java:405)
at âœ½.Added user should be displayed(file:///C:/Workspace/Workspace22052025/MyCucumberProjectV01/./Feature/002AddEmployee.feature:44)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="TestRunner.Run" name="runScenario" time="17.769"/>
  <testcase classname="TestRunner.Run" name="runScenario" time="23.264"/>
  <testcase classname="TestRunner.Run" name="runScenario" time="16.783"/>
</testsuite> <!-- TestRunner.Run -->
